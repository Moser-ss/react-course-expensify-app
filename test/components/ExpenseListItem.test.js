import React from 'react';
import { shallow } from 'enzyme';
import ExpenseListItem from '../../src/components/ExpenseListItem'
import expenses from '../data/expenses'

test('should render ExpenseListITem with expense', () => {
  const wrapper = shallow(<ExpenseListItem key={expenses[0].id} {...expenses[0]}/>)
  expect(wrapper).toMatchSnapshot()
})