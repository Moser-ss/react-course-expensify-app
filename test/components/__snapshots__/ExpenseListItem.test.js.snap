// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseListITem with expense 1`] = `
<div>
  <Link
    replace={false}
    to="/edit/qEtAP7D6Ja"
  >
    <h3>
      Gum
    </h3>
  </Link>
  <p>
    195
     - 
    0
  </p>
</div>
`;
